routes/web.php

```diff
@@ -153,4 +153,12 @@
 Route::get('/ical/event/{id}/publish', 							'IcalController@togglePublishState')->name("togglePublishState");
 
 // Lara Upgrade
-Route::get('/update',                                         'AdminController@startUpdateProcess')->name('lara.update');
\ No newline at end of file
+Route::get('/update',                                         'AdminController@startUpdateProcess')->name('lara.update');
+
+//Templates
```

Comment:
I believe this is not needed in such detail, the resource controller has some magic for this:
Simply `Route::resource('template', 'TemplateController');` should do the trick.
And rename the call to "template.overview" into "template.index" inside the views.

Reply:
tried it through development -> must duplicate code to realise the same behaviour